sass/main.scss

```diff
@@ -343,6 +343,59 @@ nav {
         }
 
     }
+    #modal {
+        display: none;
+        position: fixed;
+        z-index: 100;
+        padding-top: 27px;
+        left: 0;
+        top: 0;
+        width: 100%;
+        height: 100%;
+        overflow: auto;
+        background-color:rgba(0,0,0) ;
+        background-color: rgba(0,0,0,0.9);
```

Comment:
repetitive `background-color`?
(Also, it's a bit cleaner to use `rgba(black, 0.9)`)